src/main/java/team/creative/creativecore/common/gui/packet/ControlSyncPacket.java

```diff
@@ -24,8 +24,6 @@ public ControlSyncPacket() {}
     @Override
     public void execute(Player player, IGuiIntegratedParent container) {
         GuiSyncControl sync = GuiSyncHolder.followPathControl(path, container);
-        if (sync == null)
-            throw new RuntimeException("Could not find sync " + path);
```

Comment:
Would remove it? This can actually happen.

Reply:
Yes, because GuiSyncHolder.followPathControl already do the same check, which makes that check redundant and never fired